src/lib/Chart.tsx

```diff
@@ -92,7 +93,12 @@ function useCanvas(onResize: (sizeCpx: Size) => void) {
     const onCanvasResize = useCallback(
         function onCanvasResize(entries: ResizeObserverEntry[]) {
             if (canvas == null) return;
-            const { inlineSize: width, blockSize: height } = entries[0].devicePixelContentBoxSize[0];
+            const width =
+                entries[0]?.devicePixelContentBoxSize?.[0].inlineSize ??
+                entries[0].contentBoxSize[0].inlineSize * 2;
```

Comment:
:warning: Are you sure there should be `2`?
Shouldn't it be `devicePixelRatio`?

Reply:
Yes, you are correct! We should indeed use devicePixelRatio.

Reply:
:+1: